sf_examples/atari_examples/atari/atari_params.py

```diff
@@ -0,0 +1,52 @@
+def atari_override_defaults(env, parser):
+    """RL params specific to Atari envs."""
+    parser.set_defaults(
+        summaries_use_frameskip=False,
+        use_record_episode_statistics=True,
+        encoder_type="conv",
+        encoder_subtype="convnet_atari",
+        hidden_size=512,
+        encoder_extra_fc_layers=1,
+        obs_subtract_mean=0.0,
+        obs_scale=255.0,
+        gamma=0.99,
+        env_frameskip=4,
+        env_framestack=4,
+        exploration_loss_coeff=0.01,
+        num_workers=8,
+        num_envs_per_worker=1,
+        worker_num_splits=1,
```

Comment:
We should also do another pass at some point and provide default parameters for Atari that yield high throughput.